examples/dask/starting-dask.ipynb

```diff
@@ -111,7 +111,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "client.cluster.close()"
+    "# client.cluster.close()"
```

Comment:
Was this intended to be left commented out?

Reply:
Yeah this is intentional. I don't think we want to close the cluster when people are still running notebooks.

Reply:
I think that makes sense, but let's convert it from a comment in a code chunk to in the text have it
```
Once you are done using Dask, you can shut down the cluster using the command: client.cluster.close()`.
```

Reply:
should it be removed entirely then? Otherwise maybe there should be a comment explaining it

Reply:
@jnolis beat me to it. I like that idea ^